src/Job/SyncProject.php

```diff
@@ -136,14 +137,24 @@ public function syncProjectMedia(ScriptoProject $project)
             // a media will cascade delete all corresponding Scripto media.
             // Nevertheless, a bulk deletion will be necessary if the item/media
             // abstraction ever changes.
-            $query = $em->createQuery('
-                DELETE FROM Scripto\Entity\ScriptoMedia sm
-                WHERE sm.scriptoItem = :scripto_item_id
-                AND sm.id NOT IN (:scripto_media_ids)
-            ')->setParameters([
-                'scripto_item_id' => $sItemId,
-                'scripto_media_ids' => $sMediaIdsToRetain,
-            ]);
+            if ($sMediaIdsToRetain) {
```

Comment:
What problem are you solving with this change?

Reply:
When there are no media ids to retain (the item has only a pdf, no jpeg, and the pdf is excluded), the query is not correct : `AND sm.id NOT IN ()`.